charts/karpenter-crd/templates/karpenter.sh_nodeclaims.yaml

```diff
@@ -54,10 +54,6 @@ spec:
           name: NodeClass
           priority: 1
           type: string
-        - jsonPath: .status.conditions[?(@.type=="Drifted")].status
```

Comment:
Is this intended to be dropped?

Reply:
Sounds like this was unintentionally added and we are removing it to put it back to the state that it should be in